src/frontend-samples/transformations-sample/README.md

```diff
@@ -0,0 +1,23 @@
+# Transformations Sample
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample demonstrates how to use the Transformations API by displaying the changes in two viewports. The result of calling the transformation is being displayed side by side to the original.
+
+## Purpose
+
+The purpose of this sample is to demonstrate the following:
+
+* The usage and result of the Transformation API
+* The logic needed to balance multiple viewports.
+* How to use the Two-way viewport sync feature.
+
+## Description
+
+There are three API's used to create the transformed iModel based on a [View Definition](https://www.itwinjs.org/reference/imodeljs-backend/viewdefinitions/viewdefinition/). The first API and step, [FilterByViewDefinition](https://sbx-developer.bentley.com/api-groups/synchronization/apis/transformations/operations/filterbyviewdefinition/), is to create a transformation configuration which creates an association between source and target iModels with additional metadata to carry out the transformation. It does NOT start the transformation. 
```

Comment:
Let's update these with prod links: https://developer.bentley.com/apis/transformations/operations/filterbyviewdefinition/

Reply:
Maybe: "THere are three steps to create the transformed iModel based on a...."

Reply:
The first step is to create a transformation configuration....